plugin-server/src/config/config.ts

```diff
@@ -104,6 +104,7 @@ export function getDefaultConfig(): PluginsServerConfig {
         APP_METRICS_GATHERED_FOR_ALL: isDevEnv() ? true : false,
         MAX_TEAM_ID_TO_BUFFER_ANONYMOUS_EVENTS_FOR: 0,
         USE_KAFKA_FOR_SCHEDULED_TASKS: true,
+        UPDATE_LATEST_EVENT_CAPTURED_AT: true,
```

Comment:
might be easier to gradually roll out if this was team id list or 'everyone'. Also aren't we supposed to be able to use feature flags in the backend now?

Reply:
Maybe so but I don't really want to be testing that out on something some complex already, with the potentially to completely kill app.posthog.com.

Reply:
But maybe the standard of using an env for specific teams isn't a bad shout 💯